gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt

```diff
@@ -14,7 +14,7 @@ internal object ComposeCompilerCompatibility {
         "1.8.22" to "1.4.8",
         "1.9.0-Beta" to "1.4.7.1-beta",
         "1.9.0-RC" to "1.4.8-beta",
-        "1.9.0" to "1.5.0"
+        "1.9.0" to "1.5.1-rc01"
```

Comment:
We need to merge it to master and cherry-pick it to release/1.5.0

Reply:
changed the branch - merged to master. and cherry-picked to reelease/1.5.0